ui/pages/permissions-connect/connect-page/connect-page.tsx

```diff
@@ -244,10 +287,18 @@ export const ConnectPage: React.FC<ConnectPageProps> = ({
     [] as MergedInternalAccountWithCaipAccountId[],
   );
 
+  const evmAccountsRequested = getCaipAccountIdsFromCaip25CaveatValue(
+    requestedCaip25CaveatValue,
+  ).filter((acc) => acc.includes(KnownCaipNamespace.Eip155));
+
+  const isRequestingSpecificEvmAccounts = evmAccountsRequested.length > 0;
```

Comment:
I think we can derive whether EVM accounts are requested from existing variables in default accounts.